test/unit/routes/test-index.js

```diff
@@ -28,15 +28,28 @@ describe('routes/index', function () {
 
   describe('GET /', function () {
     it('should respond with a 200', function (done) {
-      var stubGet = sinon.stub(claims, 'get').resolves([])
       request
         .get('/')
         .expect(200)
         .end(function (error, response) {
-          expect(stubGet.calledOnce).to.be.true
           expect(error).to.be.null
           done()
         })
     })
   })
+
+  describe('GET /claims/:status', function () {
+    it('should respond with a 200', function (done) {
+      var stubClaimsList = sinon.stub(claims, 'getClaimsListAndCount').resolves({claims: [], total: {Count: 0}})
```

Comment:
names should be stupid, stub of getClaimsListAndCount should be stubGetClaimsListAndCount